spec/api/v1/customers/find_controller_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+RSpec.describe "customers::find controller" do
+  fixtures :customers
+  context "finds one instance of customer" do
+    scenario "by id" do
+      customer = customers(:one)
+      get "/api/v1/customers/find?id=#{customer.id}"
+
+      assert_response :success
+      response_customer = JSON.parse(response.body)
```

Comment:
lets think of ways to refactor these lines that are repeated throughout all of our testing.